src/signature.c

```diff
@@ -188,6 +188,7 @@ static EVP_PKEY *load_key_file(const gchar *keyfile, GError **error)
 	BIO *key = NULL;
 	unsigned long err;
 	const gchar *data;
+	gchar *passphrase;
```

Comment:
```suggestion
	const gchar *passphrase;
```

Reply:
```
src/signature.c:210:56: warning: passing argument 4 of 'PEM_read_bio_PrivateKey' discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
  210 |         res = PEM_read_bio_PrivateKey(key, NULL, NULL, passphrase);
      |                                                        ^~~~~~~~~~
```

Reply:
Too many deprecation warnings with openssl-3 :crying_cat_face:
I've taken the `g_strdup()` + `g_free()` route...

Reply:
In cases like these (where we know that OpenSSL won't write to this pointer), we've casted away const in the function call.